WordPress/gutenberg
# Fix grid layout padding on small screens

Some of the recent refactoring seems to have caused a regression in grid layout; the padding no longer adapts at certain breakpoints. This PR addresses that by moving the relevant styles to the appropriate file.

| Before | After |
| --- | --- |
| <img width="420" alt="Screenshot 2024-08-28 at 16 09 02" src="https://github.com/user-attachments/assets/7b2b4958-903f-4405-aae1-a9b1974ad89b"> | <img width="419" alt="Screenshot 2024-08-28 at 16 09 16" src="https://github.com/user-attachments/assets/32aefef0-2560-4abc-9f8b-958dd35119ca"> |



## Testing Instructions
1. Navigate to a data view like patterns or pages
2. Switch to grid layout
3. Resize the viewport
4. Notice that the grid padding adapts at the same breakpoint as other elements like filters and pagination
